lib/MetagenomeAPI/MetagenomeSearchUtils.py

```diff
@@ -16,8 +16,29 @@ def __init__(self, config):
         self.debug = config.get("debug") == "1"
         self.max_sort_mem_size = 250000
 
+    def search_contig(self, token, ref, contig_id, contig_length, start, limit):
```

Comment:
to me the name "search_contig" does not convey "find the number of features a contig has". Instead it says "find a contig". Maybe you can call this "contig_features_count"?

Reply:
yeah, the name isnt great. will change